ccm_web/client/src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { BrowserRouter as Router, Link as RouterLink, Route, Switch } from 'react-router-dom'
+import { SnackbarProvider } from 'notistack'
```

Comment:
what is the reason for choosing an external Snack bar Library? did you felt that it supported the use cases for CCM that default material-UI  did not had?

Reply:
notistack was [recommended ](https://material-ui.com/components/snackbars/#notistack)as a 'complementary'  project by material ui.

It allows multiple notifications which I believe is desirable, and you don't have to manually manage state.

Reply:
ok